packages/react-table/src/components/TableComposable/examples/ComposableTableNestedStickyHeader.tsx

```diff
@@ -0,0 +1,201 @@
+  };
+
+  // Note that we perform the sort as part of the component's render logic and not in onSort.
+  // We shouldn't store the list of data in state because we don't want to have to sync that with props.
+  let sortedConnections = connections;
+  if (activeSortIndex !== null) {
+    sortedConnections = connections.sort((a, b) => {
+      const aValue = getSortableRowValues(a)[activeSortIndex];
+      const bValue = getSortableRowValues(b)[activeSortIndex];
+      if (typeof aValue === 'number') {
+        // Numeric sort
+        if (activeSortDirection === 'asc') {
+          return (aValue as number) - (bValue as number);
+        }
+        return (bValue as number) - (aValue as number);
+      } else {
+        // String sort
+        if (activeSortDirection === 'asc') {
+          return (aValue as string).localeCompare(bValue as string);
+        }
+        return (bValue as string).localeCompare(aValue as string);
+      }
+    });
+  }
+
+  const getSortParams = (columnIndex: number): ThProps['sort'] => ({
+    sortBy: {
+      index: activeSortIndex as number,
+      direction: activeSortDirection as 'asc' | 'desc'
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+    },
+    columnIndex
+  });
+
+  return (
+    <InnerScrollContainer>
+      <TableComposable aria-label="Nested column headers table" gridBreakPoint="" isStickyHeader>
+        <Thead hasNestedHeader>
+          <Tr>
+            <Th hasRightBorder colSpan={3}>
+              {columnNames.pods}
+            </Th>
+            <Th hasRightBorder colSpan={2}>
+              {columnNames.ports}
+            </Th>
+            <Th modifier="fitContent" hasRightBorder rowSpan={2} sort={getSortParams(5)}>
+              {columnNames.protocol}
+            </Th>
+            <Th modifier="fitContent" hasRightBorder rowSpan={2} sort={getSortParams(6)}>
+              {columnNames.flowRate}
+            </Th>
+            <Th modifier="fitContent" hasRightBorder rowSpan={2} sort={getSortParams(7)}>
+              {columnNames.traffic}
+            </Th>
+            <Th modifier="fitContent" rowSpan={2} sort={getSortParams(8)}>
+              {columnNames.packets}
+            </Th>
+          </Tr>
+          <Tr>
+            <Th isSubheader sort={getSortParams(0)}>
+              {columnNames.source}
+            </Th>
+            <Th isSubheader sort={getSortParams(1)}>
+              {columnNames.destination}
+            </Th>
+            <Th isSubheader modifier="fitContent" hasRightBorder sort={getSortParams(2)}>
+              {columnNames.datetime}
+            </Th>
+            <Th isSubheader modifier="fitContent" sort={getSortParams(3)}>
+              {columnNames.source}
+            </Th>
+            <Th isSubheader modifier="fitContent" hasRightBorder sort={getSortParams(4)}>
+              {columnNames.destination}
+            </Th>
+          </Tr>
+          <Tr isBorderRow>
```

Comment:
Can you also add `aria-hidden="true"` here? Looks like doing so removes the row from total table row count.

Reply:
I'll create a followup up issue for Core.